package/demo-squeezedet-patched/patch.linux/demo.patch

```diff
@@ -0,0 +1,56 @@
+diff --git a/src/demo.py b/src/demo.py
+index 3ddafab..3b4a7de 100644
+--- a/src/demo.py
++++ b/src/demo.py
+@@ -55,7 +55,7 @@ def video_demo():
+   # out = VideoWriter(out_file_name, frameSize=(1242, 375))
+   # out.open()
+ 
+-  assert FLAGS.demo_net == 'squeezeDet' or FLAGS.demo_net == 'squeezeDet+', \
++  assert FLAGS.demo_net == 'squeezeDet' or FLAGS.demo_net == 'squeezeDet+' or FLAGS.demo_net == 'resnet50' or FLAGS.demo_net == 'vgg16', \
+       'Selected nueral net architecture not supported: {}'.format(FLAGS.demo_net)
```

Comment:
neural (also elsewhere)

Reply:
Fixed.
But it is the line from original code and changed line is only one before. Should we apply patches for all founded typos? :)